tsrelay/handler/get_serve.go

```diff
@@ -40,16 +40,22 @@ type currentTailnet struct {
 	MagicDNSEnabled bool
 }
 
+type selfStatus struct {
+	peerStatus
+	CurrentTailnet currentTailnet
```

Comment:
This should actually be analogous to the go type, and not be defined on `self` but on the top level status, because it has nothing to do with your peers or yourself, both of you are under the same tailnet.